Signal/src/call/CallService.swift

```diff
@@ -376,6 +378,8 @@ protocol CallServiceObserver: class {
         }
 
         fulfillReadyToSendIceUpdatesPromise()
+        self.fulfillReadyToSendIceUpdatesPromise = nil
```

Comment:
Yeah, perhaps its a waste of time.  PromiseKit should ensure that its safe to:

* fulfill a promise twice.
* fulfill and reject a promise, in any order.

Removed!